github/event_types.go

```diff
@@ -323,6 +323,17 @@ type OrganizationEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// OrgBlockEvent is triggered when an organization blocks or unblocks a user.
+// The Webhook event name is "org_block".
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#orgblockevent
+type OrgBlockEvent struct {
+	Action       *string       `json:"action,omitempty"`
+	BlockedUser  *User         `json:"blocked_user,omitempty"`
+	Organization *Organization `json:"organization,omitempty"`
+	Sender       *User         `json:"sender,omitempty"`
```

Comment:
You should also include `Installation` here. It's a part of all events.

```Go
...
Sender       *User         `json:"sender,omitempty"`

// The following fields are only populated by Webhook events.
Installation *Installation `json:"installation,omitempty"`
```

See https://developer.github.com/webhooks/#payloads:

> **In addition to the fields documented for each event,** webhook payloads include the user who performed the event (sender) as well as the organization (`organization`) and/or repository (`repository`) which the event occurred on, and for an Integration's webhook **may include the installation (`installation`) which an event relates to**.

Related to #505. /cc @bradleyfalzon